apps/hostd/dialogs/HostdTransactionDetailsDialog.tsx

```diff
@@ -5,11 +5,12 @@ import { useTransactions } from '../contexts/transactions'
 
 export function HostdTransactionDetailsDialog() {
   const { id, dialog, onOpenChange } = useDialog()
-  const { dataset } = useTransactions()
+  // TODO: fetch transaction so that not dependent on datasetPage.
```

Comment:
No its been like this for a long time and works fine because the only way to trigger the details dialog is when the dataset contains the specific txn, but we should still decouple this so I left a todo and added it to my list.